src/terminal/terminal.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
-
+import './terminal.scss'
```

Comment:
You forgot to include the SCSS file. After discussion, didn't need it. Will remove.